parse/grammar/KuneiformParser.g4

```diff
@@ -184,13 +216,58 @@ sql:
 
 sql_statement:
     (WITH RECURSIVE? common_table_expression (COMMA common_table_expression)*)?
-    (select_statement | update_statement | insert_statement | delete_statement)
+    (create_table_statement | alter_table_statement| create_index_statement | drop_index_statement // ddl
+    | select_statement | update_statement | insert_statement | delete_statement) // dml
 ;
 
 common_table_expression:
     identifier (LPAREN (identifier (COMMA identifier)*)? RPAREN)? AS LPAREN select_statement RPAREN
 ;
 
+create_table_statement:
+    CREATE TABLE name=identifier
+    LPAREN
+    (c_column_def | constraint_def | c_index_def)
+    (COMMA  (c_column_def | constraint_def | c_index_def))*
+    RPAREN
+;
+
+constraint_def: // TODO: rename to table_constraint
+    (CONSTRAINT name=identifier)?
+    unnamed_constraint
+;
+
+unnamed_constraint:
+    PRIMARY KEY LPAREN identifier_list RPAREN
+    | UNIQUE LPAREN identifier_list RPAREN
+    | CHECK LPAREN sql_expr RPAREN
+    | FOREIGN KEY LPAREN identifier RPAREN fk_constraint
+;
+
+alter_table_statement:
+    ALTER TABLE table=identifier
+    alter_table_action
+;
+
+alter_table_action:
+      ALTER COLUMN column=identifier SET (NOT NULL | DEFAULT literal)                  # add_column_constraint
+    | ALTER COLUMN column=identifier DROP (NOT NULL | DEFAULT | CONSTRAINT identifier) # drop_column_constraint
+    | ADD COLUMN column=identifier type                                                # add_column
+    | DROP COLUMN column=identifier                                                    # drop_column
+    | RENAME COLUMN old_column=identifier TO new_column=identifier                     # rename_column
+    | RENAME TO new_table=identifier                                                   # rename_table
+    | ADD constraint_def                                                               # add_table_constraint
+    | DROP CONSTRAINT identifier                                                       # drop_table_constraint
+;
+
+create_index_statement:
+    CREATE UNIQUE? INDEX name=identifier? ON table=identifier LPAREN  columns=identifier_list RPAREN
```

Comment:
Can you add `if not exists`?

Reply:
sure, Do we need this for other statements, like `alter table add column if not exist xxx`?  
I only create the syntax you put in the docs

Reply:
Yeah that'd be great.

Reply:
Actually I don't think this is important. As long as we have it for `create` that is ok for now. We can add it in a new release